src/core/AutomatableModel.cpp

```diff
@@ -382,13 +385,13 @@ void AutomatableModel::setAutomatedValue( const float value )
 	if( oldValue != m_value )
 	{
 		// notify linked models
-		for( AutoModelVector::Iterator it = m_linkedModels.begin();
-									it != m_linkedModels.end(); ++it )
+		for (AutoModelVector::Iterator it = m_linkedModels.begin();
+			it != m_linkedModels.end(); ++it)
 		{
-			if( (*it)->m_setValueDepth < 1 &&
-				(*it)->fittedValue( m_value ) != (*it)->m_value )
+			if (!((*it)->controllerConnection()) && (*it)->m_setValueDepth < 1 &&
```

Comment:
> However, that suggested change has a side effect I still don't know how to deal with:

We can notify the linked models in setUseControllerValue().

But before we get into the linked code, I think we need to decide whether to keep the old behavior or to touch it a minimum (even if this is buggy) to fix it later.

Reply:
I think that would do it, but you're right. I'll ask other devs what they think. Maybe it would be simpler to keep the code as you wrote it and fix the linked models behavior later.